.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Description
+
+Add a brief summary of your PR here, between one sentence and two paragraphs
+
+## Additions
+
+- Detail API additions here if any
+- Do not list changes or removals
+
+## Removals
+
+- List API removals here if any
+
+## Modifications
+
+- List changes to existing structures and functions here if any
```

Comment:
I'm thinking we shouldn't need to list these here, since they should go in the changelog

Reply:
Having it in both places can be advantageous for reviewers. Ideally it'd just be some copy pastes anyway I think.